test/unit/test_sap_cli_apirelease.py

```
@@ -0,0 +1,544 @@
+import unittest
+from unittest.mock import Mock, patch
+from argparse import ArgumentParser
+
+import sap.cli.apirelease
+import sap.cli.object
+import sap.adt
+import sap.adt.apirelease
+import sap.errors
+
+from mock import Connection, Response, BufferConsole
+from fixtures_adt_apirelease import (
```

Flake8 (7.3.0)

[error] 'fixtures_adt_apirelease.OBJECT_URI' imported but unused

(F401)